static/app/components/modals/sentryAppDetailsModal.tsx

```diff
@@ -177,7 +177,7 @@ export default class SentryAppDetailsModal extends AsyncComponent<Props, State>
                     {t('Cancel')}
                   </Button>
 
-                  <Access organization={organization} access={['org:integrations']}>
```

Comment:
Do we not need the org:integrations anymore?

Reply:
good catch